packages/next/types/index.d.ts

```diff
@@ -63,6 +63,11 @@ export type NextPage<P = {}, IP = P> = NextComponentType<NextPageContext, IP, P>
 export type PageConfig = {
   amp?: boolean | 'hybrid'
   api?: {
+    /**
+     * Flag to disable warning "API response for ${req.url} exceeds 4MB.",
+     * due to deploying app outside Vercel, like AWS.
+     */
+    bodyLimit?: boolean
```

Comment:
```suggestion
    bodyLimit?: boolean | number
```
(in bytes)

or 

```suggestion
    bodyLimit?: boolean | number | string
```
(in bytes or like `"42MB"`)

Reply:
How about something similar to the `bodyParser` config object?

Reply:
After research, it's a bit of a lift to make sure this will work correctly. Are there utils available to parse strings into bytes and bytes into strings? Don't want to reinvent the wheel, but I am not finding anything and would hate to bring in `bytes` just for this.

Reply:
Introduced a configurable object